### DISCUSSION TITLE
Impact on Wikipakk due to SharePoint Add-In Retirement?

### DISCUSSION THREAD

**Discussion by novice002** [WikiTransformationProject/wikitraccs-wikipakk-releases#18] [Q&A] [answered]
Hello!
I am trying to implement both WikiTraccs and WikiPakk as POC for now.

I recently learned about MS announcement stating that SharePoint Add-in will be retired, and will block add-ins gradually. ([LINK](https://learn.microsoft.com/en-us/sharepoint/dev/sp-add-ins/retirement-announcement-for-add-ins))

Will Wikipakk be impacted by this?

Thank you

**Comment by heinrich-ulbricht** [comment, marked as answer]
@novice002 No, WikiPakk is not impacted by the Add-In retirement. WikiPakk is a SharePoint Framework (SPFx) Solution, SharePoint Add-Ins are the predecessor of SPFx.

So, this part of the article you linked is relevant:
![image](https://github.com/WikiTransformationProject/wikitraccs-wikipakk-releases/assets/3469970/2610af96-0c4d-4873-bc98-cd9ead948961)

WikiPakk is future-proof :)

**Reply by novice002** [reply]
Thank you for the clarification! (and my apologies for not getting back to you sooner)

I am so relieved to hear that, and I can progress with peace of mind :)
It is very wise of you to have chosen SPFx way BTW.

Thank you!
